kafka/src/test/java/org/apache/apex/malhar/kafka/KafkaInputOperatorTest.java

```diff
@@ -275,10 +324,21 @@ public void testInputOperator(boolean hasFailure, boolean idempotent) throws Exc
     final LocalMode.Controller lc = lma.getController();
     lc.setHeartbeatMonitoringEnabled(false);
 
-    lc.runAsync();
```

Comment:
I added a monitor thread to make sure the Engine will be complete shutdown before start next test case